tests/PSRule.Rules.Azure.Tests/Azure.ContainerApp.Tests.ps1

```diff
@@ -139,55 +139,74 @@ Describe 'Azure.ContainerApp' -Tag 'ContainerApp' {
             $ruleResult.Length | Should -Be 3;
             $ruleResult.TargetName | Should -BeIn 'capp-B', 'capp-C', 'capp-D';
         }
-    }
 
-    Context 'Resource name - Azure.ContainerApp.Name' {
-        BeforeAll {
-            $invokeParams = @{
-                Baseline      = 'Azure.All'
-                Module        = 'PSRule.Rules.Azure'
-                WarningAction = 'Ignore'
-                ErrorAction   = 'Stop'
-            }
+        It 'Azure.ContainerApp.RestrictIngress' {
+            $filteredResult = $result | Where-Object { $_.RuleName -eq 'Azure.ContainerApp.RestrictIngress' };
 
-            $testObject = [PSCustomObject]@{
-                Name         = ''
-                ResourceType = 'Microsoft.App/containerApps'
-            }
-        }
+            # Fail
+            $ruleResult = @($filteredResult | Where-Object { $_.Outcome -eq 'Fail' });
+            $ruleResult | Should -Not -BeNullOrEmpty;
+            $ruleResult.Length | Should -Be 2;
+            $ruleResult.TargetName | Should -BeIn 'capp-B', 'capp-C';
```

Comment:
`capp-A` should fail because no rules are defined.

```suggestion
            $ruleResult.Length | Should -Be 3;
            $ruleResult.TargetName | Should -BeIn 'capp-A', 'capp-B', 'capp-C';
```